.circleci/config.yml

```diff
@@ -9,59 +9,45 @@ executors:
       - image: broadinstitute/womtool:84
         entrypoint: /bin/bash
     working_directory: ~/palantir-workflows
-    environment: WOMTOOL_JAR=/app/womtool.jar
 
 commands:
   get-cromwell-jar:
     description: "Download cromwell jar"
     steps:
       - run: wget https://github.com/broadinstitute/cromwell/releases/download/84/cromwell-84.jar
 
-  install-make:
-    description: "Install make"
+  install-watt:
+    description: "Install watt"
     steps:
       - run: |
-            apt-get update
-            apt-get install -y make
+            git clone -b ck_ignore_output https://github.com/kachulis/watt.git ~/watt
```

Comment:
change this once rickymagner/watt#2 is merged

Reply:
If you're all set making tweaks, I'd also rather make a tagged release for Watt to use here

Reply:
sounds good to me

Reply:
a couple other potential changes on my mind, but I think this is a good place for a first tagged release